README.md

```diff
@@ -1,15 +1,118 @@
-# react-lib-template
-A template that aims to make the implementation of `React` component packages easier and more methodic.
+# next-link
+
+[![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][build-status-image]][build-status-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
+
+[npm-url]:https://npmjs.org/package/@moxy/next-link
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/next-link.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/next-link.svg
+[build-status-url]:https://github.com/moxystudio/next-link/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/next-link/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/next-link
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/next-link/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/next-link
+[david-dm-image]:https://img.shields.io/david/moxystudio/next-link.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/next-link?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/next-link.svg
+
+A wrapper component that uses [next/link](https://nextjs.org/docs/api-reference/next/link) to enable client-side transitions between routes as well as external URLs.
+
+## Installation
+
+```sh
+$ npm install @moxy/next-link
+```
+
+This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
+
+## Motivation
+
+When using Next.js, we have [next/link](https://nextjs.org/docs/api-reference/next/link) to navigate between client-side routes, but when we need to navigate to external URLs an anchor tag has to be used.
+
+This component supports both, using `NextLink` we can pass an internal or external URL, using the same component for both. With the `external` prop we tell the component which type of route is and it takes care of the rest for us, rendering the `next/link` or an `<a>` tag, maintaining the same styles between the two.
```

Comment:
Just the tiniest nitpick 😅 

`With the external prop we tell the component which type of route is and it takes care of the rest for us`

I would slightly rephrase it to 

`With the external prop we specify the type of route and it takes care of the rest for us`

Just because `which type of route is` sounds a bit off. What do you think?

Reply:
I like it! Fixed.